lib/components/Buttons/ButtonBase.tsx

```diff
@@ -11,7 +11,14 @@ export type ButtonBaseProps = {
   className?: string;
   onClick?: () => void;
   href?: string;
-  variant?: "lightGray" | "gray" | "blue" | "red" | "transparent" | null;
+  variant?:
+    | "lightGray"
+    | "gray"
+    | "blue"
+    | "red"
+    | "transparent"
+    | "link"
```

Comment:
i am not quite sure why it needs another "link" variant?

If you pass in the prop `renderLink` you can pass in Next Link component with any kind of child you want.
See the prop on the ButtonBase here:
https://github.com/KlimaDAO/klimadao/blob/5be663ff72f1c48446147e7e77163125bd353533/lib/components/Buttons/ButtonBase.tsx#L42-L47

It takes the label, href and also className (the link styles you passed) from the other props.

Wouldn't this work too?

Reply:
It was to avoid needing to provide a valid href (some of these buttons were looking like links but using onclick listeners)

Either way I've removed the "link" type from ButtonBase 👍